MvpUtility/Config.cs

```diff
@@ -20,6 +20,12 @@ public class Config : IConfig
         [Description("Control over to enable or disable debug information")]
         public bool EnableDebug { get; set; } = false;
 
+        /// <summary>
+        /// Whether or not to show the MVP screen the following round start aswell
+        /// </summary>
+        [Description("Show the MVP screen next round start?")]
```

Comment:
This is not sufficient, if the player wants it only at round start, you need to add extra flag to prevent round end showing it

Reply:
Set this to true, default for this should be true, the alternative behavior is a custom choice.

Reply:
You didn't understand. Default true for EndRound, not Start.